hypothesis-python/src/hypothesis/extra/lark.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, grammar, start, explicit):
         # This is a total hack, but working around the changes is a nicer user
         # experience than breaking for anyone who doesn't instantly update their
         # installation of Lark alongside Hypothesis.
-        compile_args = getfullargspec(grammar.grammar.compile).args
+        compile_args = signature(grammar.grammar.compile).parameters
```

Comment:
I think changing this to `signature(grammar.grammar.compile, follow_wrapped=False).parameters` will preserve parity with `getfullargspec(grammar.grammar.compile).args` / will address #3245